Streamlined run task reviews: Update API changelog, add integration docs (#413)
hashicorp/terraform-docs-common: website/docs/cloud-docs/api-docs/run-tasks/run-tasks-integration.mdx

| ------------------------- | ------ | ----------------------------------------------------------------------------------------------- |
| `outcome-id`     | string | A partner supplied identifier for this outcome. |
| `description`     | string | A short, one-line, description of the result |
| `body`     | string | (Optional)  A detailed message for the result in Markdown format. |
| `url`     | string | (Optional) A URL that a user may browse to for more information about this result. |
| `tags`     | object | (Optional) An object containing tag arrays, named by the property key. |
| `tags.key`     | string |The name of the header tag. This should be a very short name ideally no more than two or three words. Special handling is given to `severity` and `status` keys. |
| `tags.key[].label`     | string | The text value of the tag. |
| `tags.key[].level`     | enum string | (Optional) The error level for the tag. Defaults to `none`. Either `none`, `info`, `warning`, or `error`. For levels other than `none`, labels are rendered with color and icon for that level. |

##### Severity and Status Tags

> When surfacing run task outcomes with tags "severity", or "status", those tags will be used to enrich the list display of outcomes in Terraform Cloud, enabling earlier feedback of severity and statuses.

Reviewer: ```suggestion
Run task outcomes with tags named "severity" or "status" are enriched within the outcomes display list in Terraform Cloud, enabling an earlier response to issues with severity and status.
```
